src/api/app/assets/stylesheets/webui2/project-package-show-grid.scss

```diff
@@ -1,28 +1,28 @@
-.package-source-files {
-  grid-area: source-files;
+.package-source-files, .project-packages {
+  grid-area: list;
 }
 
-.package-build-results {
+.package-build-results, .project-build-results {
   grid-area: build-results;
 }
 
-.package-comments {
+.package-comments, .project-comments {
   grid-area: comments;
 }
 
-.package-show {
+.package-show, .project-show {
   display: grid;
   grid-template-columns: 100%;
   grid-gap: 16px;
   grid-template-areas:
-    "source-files"
+    "list"
```

Comment:
How about file list? It's always a file list and this would be more in line with 'build-results' and 'comments'.

Reply:
It's also a package list when we are on the project show page.